css/index.css

```diff
@@ -1,3 +1,355 @@
-/* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+body {
+  line-height: 1;
+}
+ol,
+ul {
+  list-style: none;
+}
+blockquote,
+q {
+  quotes: none;
+}
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: "";
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  height: 100%;
+  font-family: "Titillium Web", sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: "Bangers", cursive;
+  letter-spacing: 1px;
+  margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+/* NavBar */
+.nav-bar {
+  display: flex;
+  width: 70%;
+  height: auto;
+  margin: 0px auto;
+}
+
+.nav-bar .nav-menu {
+  flex: 7;
+  display: flex;
+  color: rgb(204, 204, 204);
+}
+.nav-bar .nav-menu li {
+  height: 100%;
+  flex: 1;
+}
+.nav-bar .nav-menu li a {
+  text-decoration: none;
+  color: #c0c0c0;
+  display: block;
+  padding: 2rem 1.5rem;
+  margin: 0px;
+  height: 100%;
+}
+
+.nav-bar .logo-div {
+  flex: 3;
+  text-align: right;
+}
+.nav-bar .logo-div .logo {
+  padding-top: 1.25rem;
+}
+
+/* Main */
+main {
+  width: 70%;
+  margin: 0px auto;
+}
+
+/* HEADER */
+
+.main-header {
+  width: 100%;
+  border-bottom: 2px solid black;
+  display: flex;
+  padding-bottom: 50px;
+}
+.main-header .intro {
+  flex: 1;
+  padding-top: 4rem;
+}
+
+.main-header .intro span {
+  display: block;
+}
+
+.main-header .intro .intro-text {
+  font-size: 5rem;
+  font-family: "Bangers", cursive;
+  text-align: center;
+  padding-left: 2rem;
+}
```

Comment:
The padding left should be set to 0 in mobile as it is causing the text to be slightly out of line